Gemfile

```diff
@@ -33,6 +33,8 @@ end
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'pry'
+  gem 'rubocop', '~> 0.75.1', require: false
```

Comment:
Why those gems are needed here?

Reply:
That's for local linting in RubyMine in dev-time.

Reply:
Doesn't `gem install rubocop` and `rubocop %file%` do the stick?

Reply:
Hmmm... well, ok, I`ll just decline that.